Fix "Planning Scene ROS API" tutorial

sjahr opened the pull request:
### Description

Another task taken from @tylerjw's TODO list from [ros-planning/panda_moveit_config/pull/74#](https://github.com/ros-planning/panda_moveit_config/pull/74#issuecomment-703139739). I fixed the "Planning Scene ROS API" tutorial. The adaption of the object's spawn position and size also fixes the removal for me.

While the tutorial does what it should, attaching the object to the arm causes this console output:
```
ros.moveit_core.robot_state: Found empty JointState message
```
Triggered by publishing the planning diff message [here](https://github.com/ros-planning/moveit_tutorials/blob/40bd382c703ffd3dc430d90271ed4c3d1ca3213e/doc/planning_scene_ros_api/src/planning_scene_ros_api_tutorial.cpp#L184).
Any ideas why this happens?
### Checklist
- [X] Solve empty JointState message
- [X] Fix "Planning Scene ROS API" tutorial to spawn the object in the gripper and remove it from the world in the last step.
- [X] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)

JafarAbdi reviewed (commented):
Thanks @sjahr for submitting the PR!

The reason for the empty joint message error is the robot_state message should be diff

```cpp
ROS_INFO("Attaching the object to the hand and removing it from the world.");
planning_scene.world.collision_objects.clear();
planning_scene.world.collision_objects.push_back(remove_object);
planning_scene.robot_state.attached_collision_objects.push_back(attached_object);
planning_scene.robot_state.is_diff = true;
planning_scene_diff_publisher.publish(planning_scene);
```

And just for consistency, do you mind changing the frame_id for the remove_object from `panda_link0` to `panda_hand` .?

sjahr commented:
> ```c++
> planning_scene.robot_state.is_diff = true;
> ```

sjahr commented:
> Thanks @sjahr for submitting the PR!
> 
> The reason for the empty joint message error is the robot_state message should be diff
> 
> ```c++
> ROS_INFO("Attaching the object to the hand and removing it from the world.");
> planning_scene.world.collision_objects.clear();
> planning_scene.world.collision_objects.push_back(remove_object);
> planning_scene.robot_state.attached_collision_objects.push_back(attached_object);
> planning_scene.robot_state.is_diff = true;
> planning_scene_diff_publisher.publish(planning_scene);
> ```
> 
> And just for consistency, do you mind changing the frame_id for the remove_object from `panda_link0` to `panda_hand` .?

You are welcome. I've changed the frame_id for the remove_object and added the missing is_diff = true
